app/controllers/admin/userManagement.py

```diff
@@ -76,6 +78,24 @@ def updateProgramInfo(programID):
             abort(500,'Error while updating program.')
     abort(403)
 
+
+@admin_bp.route('/admin/getProgramInfo/<programID>', methods = ['GET'])
+def getProgramInfo(programID):
+    if g.current_user.isCeltsAdmin:
+        targetProgram = Program.get_by_id(programID)
+        programInfo = model_to_dict(targetProgram, recurse=False)
+        try:
+            return jsonify([programInfo])
+        except DoesNotExist:
+            flash('item not found')
```

Comment:
"item not found" is not a good error message to display to the user. Is this something that they can control?

Reply:
Also, the `Does Not Exist` error is thrown by `get_by_id()`, isn't it? not `jsonify`